### ISSUE TITLE
DEBUG Issue in my error_log file

### ISSUE THREAD

**Issue by rohit1290** [iionly/elggx_fivestar#2]
I am getting the following logs in my error_log file:

```
DEBUG: "Undefined offset: 1" in file /path/to/mod/elggx_fivestar/start.php (line 49)
DEBUG: "Undefined offset: 2" in file /path/to/mod/elggx_fivestar/start.php (line 49)
DEBUG: "Undefined index: elggx_fivestar_view" in file /path/to/mod/elggx_fivestar/start.php (line 52)
```

**Comment by rohit1290** [comment]
Solved it by adding the following line at after line 44

`if($line == NULL || $line == ""){ continue; }`

**Comment by iionly** [comment]
Do you have a view entry with an empty line? You should be able to check this in the plugin settings.

Instead of checking each line for NULL or "" values it might be an alternative to clean up the array from these entries already in the settings save action. Though even then it might be worth to also cleanup the array additionally before the foreach loop in case somone edited the views array in start.php directly.

**Comment by rohit1290** [comment]
No i dont have any view entry with an empty line.

**Comment by iionly** [comment]
I'm still trying to figure out where an empty entry might be coming from. With default fivestar views I don't get any empty lines. Have you configured any views yourself? If not, do you still have empty lines when resetting the views to default?

**Comment by rohit1290** [comment]
Yes, I am using the default value for the views. I am only using these 3 option:

```
1. group/default: elggx_fivestar_view=group/default, tag=div, attribute=class, attribute_value=elgg-subtext, after_html=<br>
2. object/album: elggx_fivestar_view=object/album, tag=div, attribute=class, attribute_value=elgg-subtext, after_html=<br>
3. object/image: elggx_fivestar_view=object/image, tag=div, attribute=class, attribute_value=elgg-subtext, after_html=<br>
```

![untitled](https://cloud.githubusercontent.com/assets/2772844/17831588/662c28aa-670a-11e6-8725-bd8382fb07e7.png)

**Comment by iionly** [comment]
Fixed in version 1.10.12 and 2.0.1.
